src/PointwiseFunctions/Hydro/EquationsOfState/DarkEnergyFluid.hpp

```diff
@@ -53,6 +53,13 @@ class DarkEnergyFluid : public EquationOfState<IsRelativistic, 2> {
                 "Dark energy fluid equation of state only makes sense in a "
                 "relativistic setting.");
 
+  std::unique_ptr<EquationOfState<IsRelativistic, 2>> get_clone()
```

Comment:
Could you add a test for the `get_clone` functions to the EOS tests? If there are some generic EOS tests then those could do that